modules/cardano-node.nix

```diff
@@ -44,6 +44,7 @@ let
        then "--bitcoin-distr \"${distributionParam}\""
        else "--flat-distr \"${distributionParam}\""))
     (optionalString cfg.jsonLog "--json-log ${stateDir}/jsonLog.json")
+    (optionalString (cfg.ekgSink != null) "--metrics +RTS -T -RTS --statsd-server ${cfg.ekgSink}")
```

Comment:
Yes, we should use `-T`, either when we invoke (as this patch does) but probably makes sense to bake it into the binary at link time by specifying `-with-rtsopts=-T` in the .cabal file.

The `-T` is needed to tell the RTS to collect GC stats, which is one of the things we report via ekg. Without that the GC numbers will all be 0.